### PR TITLE
fix: Use non-synthetic translation files

### PR DESCRIPTION
**Pull request by spydon**
The synthetic translation system is unfortunately not stable at all together with the analyzer, so this PR refactors so that we use explicitly generated l10n files (like in the other projects).

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -45,8 +45,9 @@ parts:
       
       fvm dart pub global run melos exec -c 1 --fail-fast --depends-on=build_runner -- \
       fvm dart run build_runner build --delete-conflicting-outputs
-
+
```
Comment (d-loose) [inline snap/snapcraft.yaml:48]: nitpick: some leftover whitespace

**Review by d-loose — APPROVED** [review]
LGTM :+1:

File: snap/snapcraft.yaml (hunk 1 above)
Comment (spydon) [reply snap/snapcraft.yaml]: Haha, found some more whitespace in that file (most likely added by me previously...).
I thought my editor removed trailing whitespace automagically, but apparently not for yaml files.

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -46,7 +46,11 @@ parts:
       fvm dart pub global run melos exec -c 1 --fail-fast --depends-on=build_runner -- \
       fvm dart run build_runner build --delete-conflicting-outputs
 
+      fvm dart pub global run melos exec -c 1 --fail-fast --depends-on=build_runner -- \
+      fvm dart run build_runner build --delete-conflicting-outputs
```
Comment (d-loose) [inline snap/snapcraft.yaml:50]: I suppose this got accidentally duplicated when rebasing/merging

**Pull request merged**